source/D2Common/src/D2StatList.cpp

```diff
@@ -2587,113 +1738,36 @@ BOOL __stdcall D2COMMON_10530_D2CheckStatlistFlagDMGRed(D2UnitStrc* pUnit)
 {
 	if (pUnit && pUnit->pStatListEx)
 	{
-		return pUnit->pStatListEx->dwFlags & 0x100;
+		return pUnit->pStatListEx->dwFlags & STATLIST_UNK_0x100;
 	}
 
 	return FALSE;
 }
 
 //D2Common.0x6FDB87A0 (#10532)
-int __stdcall D2Common_10532(D2StatListExStrc* pStatListEx, int nStatId)
+int __stdcall D2Common_STATLIST_GetStatUnsigned(D2StatListExStrc* pStatListEx, int nStatId)
```

Comment:
We have STATLIST_GetStatUnsigned twice now, which we should probably avoid

Reply:
Should we rename This one to something like `STATLIST_GetStatUnsigned_Layer0` or `STATLIST_GetStatUnsigned_BaseLayer` ?

Reply:
We can do that. Actually, I am wondering why it even compiles right now because one of them is exported in the .def. How does it resolve that?

Reply:
One was actually named `STATLIST_GetStatUnsigned` and the other `D2Common_STATLIST_GetStatUnsigned`, which is also a mistake I think since we most likely want the name without `D2Common_` anyway ?

Reply:
I commented on an outdated commit. For me, both are called STATLIST_GetStatUnsigned

It currently looks like this

//D2Common.0x6FDB87A0 (#10532)
int __stdcall STATLIST_GetStatUnsigned(D2StatListExStrc* pStatListEx, int nStatId)
{
	return STATLIST_GetStatUnsigned(pStatListEx, nStatId, 0);
}